backends/xnnpack/test/tester/tester.py

```diff
@@ -27,9 +29,16 @@
 from executorch.exir.passes.spec_prop_pass import SpecPropPass
 from executorch.exir.print_program import pretty_print, print_program
 
-from executorch.extension.pybindings.portable_lib import (  # @manual
-    _load_for_executorch_from_buffer,
-)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+try:
+    from executorch.extension.pybindings.portable_lib import (  # @manual
+        _load_for_executorch_from_buffer,
```

Comment:
Good point. IIRC it's used by the Serializer stage, which we (currently) do not use. It's a workaround for this issue #1648. I added the change since it blocked me from running e.g. 

`python -m unittest backends.arm.test.ops.test_add.TestSimpleAdd`

since ArmTester uses Tester => and thereby do the import.